configure.ac

```diff
@@ -4772,6 +4777,12 @@ if test "x$host" != "x$target"; then
 		sizeof_register=8
 		AC_DEFINE(TARGET_WATCHOS, 1, [...])
 		;;
+	aarch64*darwin*)
+		TARGET=ARM64
+		# Both ios and osx/arm64 have the same aarc64-darwin triple,
```

Comment:
The apple patch assumed the triple would include `darwinmacos`, right? Is that not correct?

Reply:
This is whats returned by config.guess on this machine, maybe a future version of config.guess will return a better value.

Reply:
Roger. Guess we can just update down the line if/when that changes.